### PR TITLE
allows for schedule 736 price list upload

### PR DESCRIPTION
**Pull request by tram**

### CODE REVIEW

File: calc/settings.py [hunk 1]
```diff
@@ -410,6 +411,10 @@
     'data_capture.panels.ScheduledJobsPanel',
 ]
 
+PRICE_LIST_ANALYSIS_FINDERS = [
```
Comment (hbillings) [inline calc/settings.py]: do we want to add this analysis bit to the upload, or is this left over from earlier conflation between the two?

File: data_capture/schedules/s736.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+        ][0]
+
+    def contract_model_base_year_rate(self):
+        return self.cleaned_data['price_including_iff']
+
+
+class Schedule736PriceList(BasePriceList):
+    # TODO: This class should be DRY'd out since it is nearly verbatim
+    # from the Schedule70PriceList class, but since this feature
+    # is somewhat experimental, I'm focusing more on implementation speed
```
Comment (hbillings) [inline data_capture/schedules/s736.py]: Are the comments in this file relevant, or are they copied over also?

Comment (tadhg-ohiggins) [reply data_capture/schedules/s736.py]: I think they're relevant and should be left in.

File: calc/settings.py (hunk 1 above)
Comment (tadhg-ohiggins) [reply calc/settings.py]: This is a leftover, which I'll remove.

**Comment by tadhg-ohiggins** [comment]
I decided to rebase instead of merge to bring this into line with `develop` and created a new branch and a new PR here: https://github.com/18F/calc/pull/2140

**Comment by hbillings** [comment]
Closed in favor of #2140

**Pull request closed without merging**